debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresType.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.postgresql;
+
+/**
+ * A class that binds together a PostgresSQL OID, JDBC type id and the string name of the type.
+ * The array types contain link to their element type.
+ *
+ * @author Jiri Pechanec
+ *
+ */
+public class PostgresType {
+
+    public static final PostgresType UNKNOWN = new PostgresType("unknown", -1, Integer.MIN_VALUE); 
+
+    private final String name;
+    private final int oid;
+    private final int jdbc;
+    private final PostgresType element;
+
+    public PostgresType(String name, int oid, int jdbc, PostgresType element) {
+        super();
```

Comment:
This call is superfluous.

Also I'd suggest to validate the parameters with `java.util.Objects.requireNonNull(T)` an then avoid any superflous null checks in `equals()` / `hashCode()`.